mixer/test/client/env/setup.go

```diff
@@ -139,6 +140,11 @@ func (s *TestSetup) SetStress(stress bool) {
 	s.stress = stress
 }
 
+// SetCheckDict set the checkDict flag
+func (s *TestSetup) SetCheckDict(checkDict bool) {
```

Comment:
Can you set it to `true` in at least one of mixer/test/client tests?

Reply:
So sorry! forgot to commit the new test files...